backend/src/main/java/gov/cdc/usds/simplereport/service/ApiUserService.java

```diff
@@ -213,17 +213,43 @@ private ApiUser getPatientApiUser() {
     }
   }
 
+  private static final String NOREPLY = "-noreply@simplereport.gov";
+  private static final String PATIENT_REGISTRATION_EMAIL = "patient-registration" + NOREPLY;
+
   private String getPatientIdEmail(Person patient) {
-    return patient.getInternalId() + "-noreply@simplereport.gov";
+    return patient.getInternalId() + NOREPLY;
+  }
+
+  /**
+   * The Patient Registration User should <em>always</em> exist. Originally it was created in a
+   * migration, but the app shouldn't crash if that row is missing. Instead, create it!
+   */
+  private ApiUser getPatientRegistrationApiUser() {
```

Comment:
i think we should be more consistent about saying Patient*Self*Registration. any user can register a patient, really.

Reply:
Agreed; done in 9ccce4f1